testing/correctness/apps/sequence_window/validator/validator.pony

```diff
@@ -380,7 +380,7 @@ class ReceiverFileDataSource is Iterator[Array[U8] val]
     end
 
   fun ref next(): Array[U8] val =>
-    // 4 bytes LENGTH HEADER + 8 byte U64 giles receiver timestamp
+    // 4 bytes LENGTH HEADER + 8 byte U64 data receiver timestamp
```

Comment:
data receiver has no injected timestamp. This line should remain in its old version (bearing in mind that we will probably deprecate sequence_window altogether sometime in the next few months)